src/TbCommandBar.tsx

```diff
@@ -49,7 +49,7 @@ export const TbCommandBar: React.FunctionComponent<TbCommandBarProps> = ({
                 <SearchBox
                     underlined={true}
                     placeholder="Search"
-                    onChange={(target, newValue) => onSearch(newValue)}
+                    onSearch={newValue => onSearch(newValue)}
```

Comment:
Maybe just pass {onSearch}?

Reply:
This is why is not a very good practice to code while taking care of your children.